string frame_id
string id
geometry_msgs/Vector3 grid_leaf_size
geometry_msgs/Vector3 box_dimensions
geometry_msgs/Point filter_min
geometry_msgs/Point filter_max
sensor_msgs/PointCloud2 points
---
sensor_msgs/PointCloud2 points